### PR TITLE
Update databases.md

### PR DESCRIPTION
**Pull request by dA505819**
Created Misc. Subheading for all tutorials.
Added links of tutorials.

### CODE REVIEW

File: data/databases.md [hunk 1]
```diff
@@ -26,6 +26,10 @@
 - [Time-scale](https://www.timescale.com/)
 - [kdb+](https://en.wikipedia.org/wiki/Kdb%2B) - is a column-based relational time series database (TSDB) with in-memory (IMDB) abilities, developed and marketed by [Kx Systems](https://kx.com/)
 
+## Misc.
```
Comment (neomatrix369) [inline data/databases.md:29]: You could use the title SQL as these are SQL specific links but you can do it in the next PR

**Comment by neomatrix369** [comment]
Thanks @dA505819 good work and keep them coming

File: data/databases.md (hunk 1 above)
Comment (dA505819) [reply data/databases.md:29]: SQL as a subheading can work too.

**Comment by dA505819** [comment]
@neomatrix369 Thankyou.

**Pull request merged**